src/ResourceManager/Automation/Commands.Automation/Cmdlet/RegisterAzureAutomationDscNode.cs

```diff
@@ -178,6 +178,13 @@ public string AzureVMLocation
         [PermissionSet(SecurityAction.Demand, Name = "FullTrust")]
         public override void ExecuteCmdlet()
         {
+            string parametersPassed = 
+                string.Format($"ResourceGroupName: {this.ResourceGroupName}, AutomationAccountName: {this.AutomationAccountName}, AzureVMName: {this.AzureVMName}, " +
+                $"NodeConfigurationName: {this.NodeConfigurationName}, ConfigurationMode: {this.ConfigurationMode}, ConfigurationModeFrequencyMins: {this.ConfigurationModeFrequencyMins}, " +
+                $"RefreshFrequencyMins: {this.RefreshFrequencyMins}, RebootNodeIfNeeded: {this.RebootNodeIfNeeded}, ActionAfterReboot: {this.ActionAfterReboot}, " +
+                $" AllowModuleOverwrite: {this.AllowModuleOverwrite}, AzureVMResourceGroup: {this.AzureVMResourceGroup}, AzureVMLocation: {this.AzureVMLocation}");
+            WriteDebug(parametersPassed);
+
```

Comment:
You may also want to omit ```string.Format``` method and use ```var``` instead of ```string```:
```Cs
var parametersPassed = 
    $"ResourceGroupName: {this.ResourceGroupName}, AutomationAccountName: {this.AutomationAccountName}, AzureVMName: {this.AzureVMName}, "
    + $"NodeConfigurationName: {this.NodeConfigurationName}, ConfigurationMode: {this.ConfigurationMode}, ConfigurationModeFrequencyMins: {this.ConfigurationModeFrequencyMins}, "
    + $"RefreshFrequencyMins: {this.RefreshFrequencyMins}, RebootNodeIfNeeded: {this.RebootNodeIfNeeded}, ActionAfterReboot: {this.ActionAfterReboot}, "
    + $"AllowModuleOverwrite: {this.AllowModuleOverwrite}, AzureVMResourceGroup: {this.AzureVMResourceGroup}, AzureVMLocation: {this.AzureVMLocation}";
```